src/services/request.services.js

```diff
@@ -24,6 +24,29 @@ class TripRequestService {
     const updatedRequest = Queries.updateTripRequestStatus(db.requesttrip, status, requestId);
     return updatedRequest;
   }
+
+  /** Function to search according to what the user is typing
+   * @param {integer} id the id of the logged in user
+   * @param {string} queryString the query passed in the query params
+   * @param {integer} limit the limit of the data retrieved
+   * @param {integer} offset the offset to know how to divide pages
+   * @returns {object} found data
+   */
+  static async searchFunction(id, queryString, limit, offset) {
```

Comment:
Hey @jo-rdan, nice how on this one. However, I think to name this function `search` was enough. let me know what do you think.